src/Cmfd.cpp

```diff
@@ -1140,23 +1138,23 @@ void Cmfd::getSurfaceDiffusionCoefficient(int cmfd_cell, int surface,
                / (delta_next * dif_coef + delta * dif_coef_next);
 
     /* Compute the surface-averaged net current across the surface */
-    current = sense * (current_out - current_in) / delta_interface;
+    current = (current_out - current_in) / delta_interface;
 
     /* Compute the surface diffusion coefficient correction */
-    dif_surf_corr = -(sense * dif_surf * (flux_next - flux) + current)
+    dif_surf_corr = -(dif_surf * (flux_next - flux) + current)
         / (flux_next + flux);
 
     /* Flux limiting condition */
     if (_flux_limiting && _moc_iteration > 0) {
       double ratio = dif_surf_corr / dif_surf;
       if (std::abs(ratio) > 1.0) {
 
-        if (sense * current > 0.0)
```

Comment:
well there s the reason sense is here
This seems to be a test to see if it's outgoing or not. 

You can remove sense from dif_surf_corr but not from dif_surf imo

Reply:
Oh duh - yeah that makes sense. I'll add back.

Reply:
nah you're right it cancels out here too.